pkg/events/events.go

```diff
@@ -0,0 +1,77 @@
+package events
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/record"
+)
+
+// Event message format "medik8s <operator shortname> <message>"
+const customFmt = "[%3s] %s"
+
+type EventRecorder interface {
+	// NormalEventf will record an event with type Normal and fixed message.
+	NormalEvent(object runtime.Object, reason, message string)
+	// NormalEventf will record an event with type Normal and formatted message.
+	NormalEventf(object runtime.Object, reason, messageFmt string, a ...interface{})
+	// WarningEventf will record an event with type Warning and fixed message.
+	WarningEvent(object runtime.Object, reason, message string)
+	// WarningEventf will record an event with type Warning and formatted message.
+	WarningEventf(object runtime.Object, reason, messageFmt string, a ...interface{})
+
+	// Event overrides the default event recorder to add the operator shortname to the message.
+	Event(object runtime.Object, eventtype, reason, message string)
+	// Eventf overrides the default event recorder to add the operator shortname to the message.
+	Eventf(object runtime.Object, eventtype, reason, messageFmt string, args ...interface{})
+	// AnnotatedEventf overrides the default event recorder to add the operator shortname to the message.
+	AnnotatedEventf(object runtime.Object, annotations map[string]string, eventtype, reason, messageFmt string, args ...interface{})
+}
+
+type eventRecorder struct {
+	record.EventRecorder
+	operator string
+}
+
+var _ EventRecorder = &eventRecorder{}
```

Comment:
hmm I missing something, why is this needed  ? 🤔

Reply:
Not needed in fact 🫣